TAScheduler/UnitTests/tests.py

```diff
@@ -76,71 +77,879 @@ class TestAdminCourseInstrAsgmt(TestCase):  # Kevin
 class TestAdminCourseTAAsgmt(TestCase):  # Kevin
     pass
 
-class TestTAInit(TestCase):
+
+class TestTAInit(TestCase):  # Kevin
     pass
 
+
+# I ORIGINALLY INTERPRETTED:
+# "instructor max assignments" as max "course" assignments for an instructor &
+# "ta max assignments" as max "lab" assignments for a TA ... I think it might be better to just make it "max course assignments" for both?
 class TestTAHasMaxAsgmts(TestCase):  # Kiran
-    pass
+    taDB = None
+    courseDB = None
+    user = None  # for TA
+    taObj = None
+    adminObj = None  # for deleting course
+
+    def setUp(self):
+        self.user = User.objects.create(
+            email_address='TA@example.com',
+            password='TApassword',
+            first_name='TA',
+            last_name='User',
+            home_address='123 TA St',
+            phone_number='1234567890'
+        )
+        self.taDB = TA.objects.create(user=self.user, max_assignments=1)  # max 1 assignment!
+        self.taObj = TAObj(self.taDB)  # creating TA object using TA in database.
+        tempUserForAdmin = User(email_address="admin@example.com")
```

Comment:
I believe the database will throw an error if you try to make a user with only an email. It expects all fields to be provided.